src/components/Profile/My posts/MyPosts.jsx

```diff
@@ -24,7 +24,7 @@ const MyPosts = ({ posts, addPost }) => {
       <StyledText>{getPostElements()}</StyledText>
```

Comment:
Bad naming - should be not a "StyledText". You map other components inside this block and in these other components, you have maybe another one... And after it, this container named just "Text"?